free-science-books.md

```diff
@@ -113,9 +118,15 @@
 * [A Dictionary of Chemistry (1777)](https://unglue.it/work/140195/) - Pierre Joseph Macquer and James Keir
 * [Biochemistry from a phenomenological point of view](http://www.louisbolk.org/downloads/1282.pdf) - Christa van Tellingen (PDF)
 * [Biochemistry Practice Book](http://www.ets.org/s/gre/pdf/practice_book_biochem.pdf) - (PDF)
+* [Chemistry Overview | English](https://web.ung.edu/media/Chemistry2/Chemistry-LR.pdf) (PDF)
 * [CK-12 Chemistry - Second Edition (2011)](https://unglue.it/work/140345/) - Sharon Bewick et al.
 * [From Vitamins to Baked Goods: Real Applications of Organic Chemistry (2012)](https://unglue.it/work/145870/) - Kieran Gallagher and Lizhi Fan
+* [General Chemistry | English](https://www.wiley.com/college/sc/klein/doc/c01_AReviewOfGeneralChemistry_Klein.pdf) (PDF)
```

Comment:
This is just chapter one of a book, presented without author or the actual title. In general, we don't like these.